transfer/src/main/java/org/kinecosystem/transfer/receiver/view/IAccountInfoView.java

```diff
@@ -7,6 +7,5 @@
 
     void close();
 
-    void updateSourceApp(String sourceApp);
-
+    void updateTransactionInfo(String senderAppName, String memo, String senderAppId,  String receiverAppId);
```

Comment:
can we make a base empty implementation of this method so that it is not compulsory to implement it ?